Tests/Functional/TestBase.php

```diff
@@ -154,4 +163,61 @@ protected function compareRecords($expectedRecords, $actualRecords, $checkAllFie
 
         $this->assertEquals($expectedRecords, $actualRecords);
     }
+
+    /**
+     * Executes an SQL file.
+     *
+     * @param Connection $conn
+     * @param string     $file
+     */
+    protected function executeSqlFile(Connection $conn, $file)
+    {
+        $sql = file_get_contents($file);
+        $stmt = $conn->prepare($sql);
+        $stmt->execute();
+    }
+
+    /**
+     * Executes large SQL file.
+     *
+     * @param string $filename
+     */
+    protected function executeLargeSqlFile($filename)
```

Comment:
I would probably propose getting rid of one of executeSqlFile and executeLargeSqlFile. Maybe keep the logic from executeLargeSqlFile, but use executeSqlFile(Connection $conn, $file)?